### ISSUE TITLE
Single page view to release notes

### ISSUE THREAD

**Issue by anuraaga** [line/armeria#3957]
The Armeria release notes only show a single release per page - this makes the "find a behavior difference and go through releases to find the change" workflow quite tedious. GitHub is a little better by providing pagination with multiple releases per view, Armeria's docs page could possibly be even better using an infinite scroll view.

**Comment by trustin** [comment]
That's a good suggestion, @anuraaga. Would adding a separate page that includes all release notes in a single page work?

**Comment by anuraaga** [comment]
Yeah if it's linked from the smaller pages too then should be good.
